src/test/java/org/llorllale/cactoos/matchers/HasValuesTest.java

```diff
@@ -67,18 +67,29 @@ public void contains() {
      *  The possible name - {@code IsMatcherOf}.
      */
     @Test
-    public void containsErrorMessage() {
-        final Description msg = new StringDescription();
+    public void thatErrorMessageForStartWithIsCorrect() {
+        final Description expected = new StringDescription();
```

Comment:
@dgroup the naming of these two variables - `expected` and `actual` - is confusing when taking into account the way you're using them. You testing `expected` for some reason... 

In a test, I expected a variable called `expected` to be initialized *without* the use of the object under test, and the `actual` variable to be compared against `expected`.

Reply:
@llorllale , agree, done.